Makefile

```diff
@@ -78,4 +86,5 @@ install.tools: .install.gitvalidation .install.gometalinter
 	help \
 	install \
 	lint \
-	uninstall
+	uninstall \
+        version
```

Comment:
Odd. I don't see this in my editor and when I go to the file in Github, I don't see the space.